crates/core/tedge_watchdog/src/dummy_watchdog.rs

```diff
@@ -0,0 +1,3 @@
+pub async fn start_watchdog(_config_dir: PathBuf) -> Result<(), anyhow::Error> {
+    // nothing here, as this is a dummy.
```

Comment:
I think you're missing something to return here ;)

Reply:
Whoopsi.